go/control/api/api.go

```diff
@@ -156,6 +166,12 @@ type RuntimeStatus struct {
 	Storage *storageWorker.Status `json:"storage,omitempty"`
 }
 
+// SeedStatus is the status of the seed node and its services.
+type SeedStatus struct {
+	// Tendermint is the Tendermint seed node service status.
+	Tendermint *tmSeed.Status `json:"tendermint,omitempty"`
```

Comment:
I would make this independent of the consensus layer backend, maybe even move the fields here directly. There is probably no need to split this up too much.

Reply:
This is true, but the existence of multiple internal services should be an internal detail as it may change in the future (e.g. migrate towards not having this Tendermint-specific service).